### DISCUSSION TITLE
Toggling Feature Flags through SDK's

### DISCUSSION THREAD

**Discussion by Aethen22** [Unleash/unleash#2882] [Q&A] [answered]
Hello there, kind of a newbie in CI/CD territory and i've been testing Unleash, both on K8's and docker, and i didnt see any way to change the state of a Feature Toggle(from enabled to disabled or viceversa), so the question is, is there a way without using a third-party application? like maybe the API or some hidden function in the SDK's(Specifically python)?

Thanks in advance!

**Comment by ivarconr** [comment, marked as answer]
Hi there, the SDK is only supposed to be used to consume feature toggle, and evaluate their status for a given context.

Unleash do however [document the full Admin API with Open API ](https://docs.getunleash.io/reference/api/unleash)and it should be fairly simple to call the API directly (or generate a client based on the specification). 

E.g. to enable a toggle in an environment you can simply POST to the [toggleFeatureEnvironmentOn action on the features API](https://docs.getunleash.io/reference/api/unleash/toggle-feature-environment-on):

```
POST /api/admin/projects/:projectId/features/:featureName/environments/:environment/on
```

**Reply by Aethen22** [reply]
Thanks a lot, was checking the API documentation but didnt find it, exactly the answer i was looking for!
